packages/vaadin-combo-box/test/combo-box-light.test.js

```diff
@@ -1,13 +1,12 @@
 import { expect } from '@esm-bundle/chai';
 import sinon from 'sinon';
-import { fixtureSync } from '@open-wc/testing-helpers';
+import { fire, click, enter, fixtureSync, mousedown, mouseup, touchend, touchstart } from '@vaadin/testing-helpers';
 import { resetMouseCanceller } from '@polymer/polymer/lib/utils/gestures.js';
-import { downAndUp, pressAndReleaseKeyOn } from '@polymer/iron-test-helpers/mock-interactions.js';
 import { PolymerElement, html } from '@polymer/polymer/polymer-element.js';
 import '@polymer/iron-input/iron-input.js';
 import '@polymer/paper-input/paper-input.js';
 import '@vaadin/vaadin-text-field/vaadin-text-field.js';
-import { createEventSpy, fire, TOUCH_DEVICE } from './helpers.js';
+import { createEventSpy, isSafari, TOUCH_DEVICE } from './helpers.js';
```

Comment:
Consider using [`isDesktopSafari`](https://github.com/vaadin/testing-helpers/blob/a42f51c3218326ba5bb10ec1ed2e1450a8beca26/src/browser.ts#L5-L6) helper from `@vaadin/testing-hepers` here, it should cover WebKit.

Reply:
Thanks! Fixed.